proto/runtime_config_linux.proto

```diff
@@ -0,0 +1,213 @@
+package oci;
+
+// LinuxStateDirectory holds the container's state information
+message DefaultState {
+  // TODO(vbatts) not as elegant in some ways, but there is not a concept of const here
+  optional string Directory = 1 [default =  "/run/opencontainer/containers"];
+}
+
+// LinuxRuntime hosts the Linux-only runtime information
+message LinuxRuntime {
+	// UIDMapping specifies user mappings for supporting user namespaces on linux.
+  repeated IDMapping UIDMapping = 1;
```

Comment:
i've got a commit staged, and it will change up a lot of the go style.
Like `GID` -> `gid` -> `Gid`